docs/vectorizer-quick-start.md

```diff
@@ -124,7 +123,7 @@ To create and run a vectorizer, then query the auto-generated embeddings created
     ```sql
     SELECT
         chunk,
-        embedding <=>  ai.openai_embed('text-embedding-3-small', 'pgai', _dimensions=>768) as distance
+        embedding <=>  ai.openai_embed('text-embedding-3-small', 'pgai', dimensions=>768) as distance
```

Comment:
👍 

For next reviewers: https://github.com/timescale/pgai/blob/fe84454837007f6ad61b2114e79adbffd64ddc6d/projects/extension/sql/idempotent/001-openai.sql#L63